installer/conf/sudoers

```diff
@@ -41,6 +41,9 @@ omsagent ALL=(ALL) NOPASSWD: /opt/microsoft/omsconfig/Scripts/OMSAuditdPlugin.sh
 # run omsbaseline
 omsagent ALL=(ALL) NOPASSWD: /opt/microsoft/omsagent/plugin/omsbaseline *
 
+# run process investigator
```

Comment:
what is the context of this change? why does this needs sudoers access?

Reply:
can you please send a mail to OMSLinuxAgent@microsoft.com explaining the need for this change?